### PR TITLE
Fixed #25196 -- Corrected test db name in "Destroying old test database" message

### PR DESCRIPTION
**Pull request by darkryder**
Ticket: https://code.djangoproject.com/ticket/25196

### CODE REVIEW

File: django/db/backends/base/creation.py [hunk 1]
```diff
@@ -47,7 +47,7 @@ def create_test_db(self, verbosity=1, autoclobber=False, serialize=True, keepdb=
                 action = "Using existing"
 
             print("%s test database for alias '%s'%s..." % (
-                action, self.connection.alias, test_db_repr))
+                action, test_database_name, test_db_repr))
```
Comment (timgraham) [inline django/db/backends/base/creation.py]: This change isn't correct. The test database name isn't the alias.

File: django/db/backends/base/creation.py [hunk 2]
```diff
@@ -174,7 +174,7 @@ def _create_test_db(self, verbosity, autoclobber, keepdb=False):
                     try:
                         if verbosity >= 1:
                             print("Destroying old test database '%s'..."
```
Comment (timgraham) [inline django/db/backends/base/creation.py]: I think the idea of Shai's comment on the other PR was to change this message to say "Destroying old test database for alias '%' (test_db_name)" similar to the other messages and to use a helper function to generate it so it's less error prone.

**Comment by timgraham** [comment]
Since you deassigned yourself on the ticket, I guess that means you don't plan to continue work?

**Comment by darkryder** [comment]
No, I don't think I'd be able to right now.

**Pull request closed without merging**